north/src/runtime/error.rs

```diff
@@ -29,6 +29,8 @@ pub enum Error {
     MissingResource(String),
     #[error("Container {0} already installed")]
     ContainerAlreadyInstalled(String),
+    #[error("Repository: {0}")]
+    Repository(String),
```

Comment:
There are only 2 error conditions with a repository:
- not found
- not writable
I would make those 2 distinct errors without a `String`